h/views/organizations.py

```diff
@@ -1,8 +1,12 @@
+from pyramid.exceptions import NotFound
 from pyramid.view import view_config
 
 
 @view_config(
     route_name="organization_logo", request_method="GET", renderer="svg", http_cache=600
 )
-def organization_logo(logo, request):
-    return logo
+def organization_logo(organization, request):
+    if not organization.logo:
+        raise NotFound()
```

Comment:
Slightly odd but apparently `pyramid.NotFound` is just an alias of `pyramid.httpexceptions.HTTPNotFound`

Reply:
Is there any difference / preference between the two?

Reply:
No, I think it's a literal alias